site/docs/versions.html

```diff
@@ -5,8 +5,9 @@
 ---
 
 <div class="row">
-{% for release in site.data.docs-versions %}
-  <div class="col-md">
+{% assign releases = site.data.docs-versions | reverse %}
```

Comment:
This change is different than #31754

Reply:
NVM, I saw the the OP. maybe change the commit comment to reflect this?

Reply:
PR reflects it, that good enough? Can also edit it when squashing and merging.

Reply:
The referenced PR doesn't include the reverse change, hence why I suggest to edit the commit message. Also, please do not squash when merging; these patches should be separate commits since they refer to different changes :)